tests/certification/pubsub/pulsar/docker-compose.yml

```
@@ -0,0 +1,19 @@
+version: '2'
+services:
+  standalone:
+    image: apachepulsar/pulsar:2.8.2
+    ports:
+      - "6650:6650"
+      - "8080:8080"
+    environment:
+      - BOOKIE_MEM=" -Xms768m -Xms768m -XX:MaxDirectMemorySize=1g"
```

```suggestion
      - BOOKIE_MEM=" -Xms768m -Xms768m -XX:MaxDirectMemorySize=1g"
```
- BOOKIE_MEM=" -Xms768m -Xmx768m -XX:MaxDirectMemorySize=1g"